CONTRIBUTING.md

```diff
@@ -35,6 +35,16 @@ Navigate into the repo:
 cd pynecone
 ```
 
+Install poetry:
+```
+pip install poetry
```

Comment:
I think this should be `curl -sSL https://install.python-poetry.org | python3 -`
Based on: https://python-poetry.org/docs/

Reply:
Great idea